### ISSUE TITLE
[Coolify won't remove the Docker container when deleting from dashboard]: 

### ISSUE THREAD

**Issue by EgloDevelopment** [coollabsio/coolify#800]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Example public repository

coolify

### Description

When you delete a container from the dashboard "Such as Appwrite" it is not removed on Docker, and it remains active, even on reboot.

### Steps To Reproduce

Create new app, (Eg: Pocketbase)
Delete app from dashboard
Check Docker and app is still there

### Version

3.12.2

**Comment by ChikoShidori** [comment]
Are you sure? I did it the same way you say it happens
![image](https://user-images.githubusercontent.com/53100578/208659520-50e93d3a-0523-4aab-8243-7f66bc6f1705.png)

Maybe you are confusing images with containers (the one created 4 minutes ago is the one I used to do the test)
![image](https://user-images.githubusercontent.com/53100578/208659733-26567681-4958-431e-b4db-dff4cb3942e9.png)

**Comment by andrasbacsai** [comment]
Are you using a remote docker engine?

**Comment by Mateleo** [comment]
I have the same problem. The app is deleted from coolify and is not visible by docker anymore. But the app/folders is not removed from disk. Even with "sudo docker system prune" the app/folder is not deleted.

**Comment by b1mind** [comment]
Same issue with a WordPress/SQL service deploy, hit the trashcan from the dashboard for the first time and now realize it did not remove from disk and cleanup does not remove it either.

**Comment by themarkwill** [comment]
I have the same problem, in my case I use multiples network, and I have this problem.

**Comment by piscis** [comment]
Happened to me today I installed a couple of apps and services and then deleted them from the dashboard without stopping the result was that the corresponding containers where still running but the entry from the app was deleted. I had to manually figure out which containers should have been removed and then deleted them manually. AFAIK stoping the app or service and then delete it from the „danger zone“ section inside the apps configuration works and removes the containers.
